roles/technical_principal.md

```diff
@@ -0,0 +1,154 @@
+
+We expect our commercially aware engineers to quickly become a trusted partner to the customer team - both our day-to-day contacts, and more senior stakeholders. A commercially aware engineer should be able to put a customer at ease by engaging in small talk, and by building strong and friendly working relationships. A commercially aware engineer is expected to credibly and knowledgeably represent Made Tech when conversing with senior stakeholders.
+
+### To proactively identify opportunities where we can widen the impact of our work with the customer organisation
+
+We expect our commercially aware engineers to proactively seek opportunities where we can extend our remit with the customer organisation. A commercially aware engineer is expected to understand the objectives of our stakeholders, and to sell Made Tech’s wider capabilities.
+
+### To make sensible, well reasoned commercial decisions
+
+We expect our commercially aware engineers to proactively make good commercial decisions. A commercially aware engineer is expected to identify and course correct potential delivery issues before they become impactful. A commercially aware engineer is expected to proactively escalate larger delivery issues to the Delivery Manager or Delivery Director.
+
+## Senior Engineer Expectations
+
+### To make sensible and well justified technical architecture decisions, involving the team in the decision making process where appropriate
+
+We expect our Senior Engineers to be able to design and implement appropriate technical architectures to solve problems. We expect them to appropriately involve the team in the decision making process to help them develop their technical architecture skills.
+
+### To have attained the Made Tech Core Skills
+
+We expect our Senior Engineers to be an expert in their craft, practising and coaching others in the Made Tech Core Skills
+
+### To be willing and able to pick up new programming languages, technologies and techniques
+
+We expect our Senior Engineers to be actively researching and trialling new technologies and techniques. We expect them to be regularly sharing their knowledge whether that is during focussed learning sessions, talks, or via pairing.
+
+### To be leading the way with introducing new technologies to deliveries
+
+We expect our Senior Engineers to help teams use new technologies as part of deliveries. We expect them to make suggestions before new projects boot up and to help the adoption of the technology during the delivery through pairing and otherwise coaching their colleagues.
+
+### To advocate lean and iterative approaches to solving problems
+
+We expect our Senior Engineers to always push for solving a problem with the simplest, most sensible solution rather than taking a more over-engineered approach. We expect them to encourage an early and often approach for getting functionality into production.
+
+## Core Engineer Expectations
```

Comment:
I have a question on how relevant these are at the TP level. I think this is a different set of skills over being good at hands-on TDD.

Reply:
Have removed Senior and Core expectations.